src/c++/perf_analyzer/load_manager.cc

```diff
@@ -143,6 +143,20 @@ LoadManager::ResetIdleTime()
   }
 }
 
+const size_t
+LoadManager::GetNumSentRequests()
```

Comment:
should this be GetAndResetNumSentRequests()?

Reply:
Renamed 👍